client/my-sites/domains/domain-management/list/list-all.jsx

```diff
@@ -48,8 +63,9 @@ class ListAll extends Component {
 			<Main wideLayout>
 				<FormattedHeader headerText={ translate( 'All Domains' ) } align="left" />
 				<div className="list-all__container">
-					{ config.isEnabled( 'manage/all-domains' ) && <QueryAllDomains /> }
-					{ this.props.sites.map( ( site, index ) => this.renderSingleSite( site, index ) ) }
+					{ config.isEnabled( 'manage/all-domains' )
```

Comment:
I don't think we need `manage/all-domains` feature flag as this view is only shown if the `manage/all-domains` is enabled anyway